ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml

```
@@ -2,6 +2,9 @@ ref:
   as: hypershift-hostedcluster-create-hostedcluster
   cli: latest
   env:
+  - name: PLATFORM
+    default: "aws"
+    documentation: "The platform(AWS/PowerVS) of the HostedCluster. AWS by default"
```

lets document the exact the options we are using
```suggestion
    documentation: "The platform(aws/powervs) of the HostedCluster. aws by default"
```

---

Done